lib/i18n/activities.js

```diff
@@ -12,109 +22,223 @@ export const ActivityDescriptionI18n = defineMessages({
   },
+  COLLECTIVE_VIRTUAL_CARD_MISSING_RECEIPTS: {
+    defaultMessage: 'Notified admins about a missing receipt for <Expense>expense</Expense> on <Account></Account>',
+  },
+  COLLECTIVE_VIRTUAL_CARD_SUSPENDED: {
+    defaultMessage: 'Virtual card suspended on <Account></Account>',
+  },
+  VIRTUAL_CARD_REQUESTED: {
+    defaultMessage: 'Requested a virtual card for <Account></Account>',
+  },
+  VIRTUAL_CARD_CHARGE_DECLINED: {
+    defaultMessage: 'virtualcard.charge.declined',
+  },
+  // Connected accounts
   CONNECTED_ACCOUNT_CREATED: {
     id: 'WebhookEvents.CONNECTED_ACCOUNT_CREATED',
     defaultMessage: 'Connected account added',
   },
+  // Contributions
   SUBSCRIPTION_CANCELED: {
     defaultMessage: 'Subscription for <Order>order</Order> cancelled',
   },
   TICKET_CONFIRMED: {
     id: 'WebhookEvents.TICKET_CONFIRMED',
     defaultMessage: 'Ticket confirmed',
   },
-});
-
-export const ActivityCategoryLabelI18n = defineMessages({
-  ALL: {
-    id: 'Amount.AllShort',
-    defaultMessage: 'All',
+  CONTRIBUTION_REJECTED: {
+    defaultMessage: 'Contribution from <FromAccount></FromAccount> rejected by <Account></Account>',
+  },
+  ORDER_CANCELED_ARCHIVED_COLLECTIVE: {
+    defaultMessage: '<Order>Order</Order> cancelled on <Account></Account> (archived)',
+  },
+  ORDER_PROCESSING: {
+    defaultMessage: '<Order>Order</Order> from <FromAccount></FromAccount> to <Account></Account> set as processing',
+  },
+  ORDER_PROCESSING_CRYPTO: {
+    defaultMessage:
+      'Crypto <Order>order</Order> from <FromAccount></FromAccount> to <Account></Account> set as processing',
+  },
+  ORDER_PENDING_CONTRIBUTION_NEW: {
+    defaultMessage: 'New pending <Order>order</Order> from <FromAccount></FromAccount> to <Account></Account>',
+  },
+  ORDER_PENDING_CONTRIBUTION_REMINDER: {
+    defaultMessage: 'Sent reminder to <FromAccount></FromAccount> about <Order>order</Order> to <Account></Account>',
+  },
+  BACKYOURSTACK_DISPATCH_CONFIRMED: {
+    defaultMessage: 'BackYourStack dispatch confirmed for <Account></Account>',
+  },
+  PAYMENT_FAILED: {
+    defaultMessage: 'Payment from <FromAccount></FromAccount> to <Account></Account> for <Order>order</Order> failed',
+  },
+  PAYMENT_CREDITCARD_CONFIRMATION: {
+    defaultMessage: 'Asked for credit card confirmation for <Order>order</Order> on <Account></Account>',
+  },
+  PAYMENT_CREDITCARD_EXPIRING: {
+    defaultMessage: 'Sent a reminder about expiring credit card to <Account></Account>',
+  },
+  USER_PAYMENT_METHOD_CREATED: {
+    defaultMessage: 'Created a new payment method for <Account></Account>',
+  },
+  // Sign in
+  USER_NEW_TOKEN: {
+    defaultMessage: 'Requested a new sign in token',
+  },
+  OAUTH_APPLICATION_AUTHORIZED: {
+    defaultMessage: 'Authorized a new OAuth application',
   },
-  COLLECTIVE: {
-    defaultMessage: 'Accounts',
+  USER_CHANGE_EMAIL: {
+    defaultMessage: 'Changed email address',
   },
```

Comment:
@Betree : For `USER_NEW_TOKEN` and `USER_CHANGE_EMAIL` shouldn't we specify which user changed the email address? Especially if we are showing the activity log in the root actions section. 🤔 And for `OAUTH_APPLICATION_AUTHORIZED`, the name of the oauth application?

Reply:
>  For USER_NEW_TOKEN and USER_CHANGE_EMAIL shouldn't we specify which user changed the email address?

That's already the case:

![image](https://user-images.githubusercontent.com/1556356/187189802-98a5bc33-0871-44c4-a113-8f779429e771.png)

> And for OAUTH_APPLICATION_AUTHORIZED, the name of the oauth application?

Would be nice to have that indeed, though it requires some API changes.